.github/actions/node-setup/action.yaml

```diff
@@ -8,10 +8,10 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Setup Node.js 18.x
+    - name: Setup Node.js 20.x
       uses: actions/setup-node@v3
       with:
-        node-version: 18.x
+        node-version: 20.x
```

Comment:
I am not sure if it officially supports it 🥲 

The docs (https://developers.cloudflare.com/workers/wrangler/install-and-update/#install-wrangler) say Nodejs 16.17.0 and later:
![Screenshot 2024-01-16 at 15 45 04](https://github.com/cloudflare/next-on-pages/assets/61631103/ca0d7d43-f323-4040-bb96-57853cb49c9c)

(since 16 is deprecated I wonder if we should update that 🤔)

But I do need to use node 20 because of this: https://github.com/cloudflare/workers-sdk/issues/4563#issuecomment-1862541486

What do you think? 😓

Reply:
Let's go ahead with node 20 here. The case where node 18 fails is somewhat unusual I think in everyday usage.